x/ibc/03-connection/keeper/handshake.go

```diff
@@ -48,7 +48,7 @@ func (k Keeper) ConnOpenTry(
 	connectionID string, // desiredIdentifier
 	counterparty types.Counterparty, // counterpartyConnectionIdentifier, counterpartyPrefix and counterpartyClientIdentifier
 	clientID string, // clientID of chainA
-	counterpartyVersions []string, // supported versions of chain A
+	counterpartyVersions []types.Version, // supported versions of chain A
```

Comment:
```suggestion
	counterpartyVersions []types.VersionI, // supported versions of chain A
```

Reply:
I assume this is supposed to be `exported.VersionI`?